server/src/main/java/io/crate/expression/symbol/ScopedSymbol.java

```diff
@@ -129,11 +148,12 @@ public int hashCode() {
         int result = relation.hashCode();
         result = 31 * result + column.hashCode();
         result = 31 * result + dataType.hashCode();
+        result = 31 * result + Objects.hashCode(symbol);
         return result;
     }
 
     @Override
     public long ramBytesUsed() {
-        return dataType.ramBytesUsed();
+        return dataType.ramBytesUsed() + (symbol == null ? 0 : symbol.ramBytesUsed());
```

Comment:
This method hints the RAM usage of a row value for this column, it's not related to the memory usage of the original symbol. So this change should be dropped.